releases/4.7.0-draft.md

```diff
@@ -11,48 +11,142 @@ TurfJS releases are being tracked using [Milestones](https://github.com/Turfjs/t
+- Adds new `isNumber` function and improves type checking for few [`@turf/helpers`][helpers] methods
 (PR https://github.com/Turfjs/turf/pull/920 - Contributor @DenisCarriere @stebogit)
-- Adds throw errors to invalid `MultiPolygons` for `@turf/simplify`
+- Adds throw errors to invalid `MultiPolygons` for [`@turf/simplify`][simplify]
 (PR https://github.com/Turfjs/turf/pull/922 - Contributor @DenisCarriere)
 
 
 ## 🐛 Bug Fixes
-- Fixes [`@turf/bbox-clip`](https://github.com/Turfjs/turf/tree/master/packages/turf-bbox-clip) point intersection handling, adding sanity-checks the lineclip output
+- Fixes [`@turf/bbox-clip`](bbox-clip) point intersection handling, adding sanity-checks the lineclip output
 (PR https://github.com/Turfjs/turf/pull/886)
-- Fixes `@turf/line-split` endpoint case handling applying `@turf/truncate` on `splitter`
+- Fixes [`@turf/line-split`][line-split] endpoint case handling applying [`@turf/truncate`][truncate] on `splitter`
 (PR https://github.com/Turfjs/turf/pull/892 and https://github.com/Turfjs/turf/pull/855 - Contributor @stebogit)
-- Fixes `@turf/intersect` throwing "uncaught exception", adding `@turf/truncate` to inputs and returning `Feature<null>` if no geometry
+- Fixes [`@turf/intersect`][intersect] throwing "uncaught exception", adding [`@turf/truncate`][truncate] to inputs and returning `Feature<null>` if no geometry
 (PR https://github.com/Turfjs/turf/pull/890 - Contributor @stebogit @vicvolk)
-- Fixes `@turf/hex-grid` not properly cloning the last ring vertex
+- Fixes [`@turf/hex-grid`][hex-grid] not properly cloning the last ring vertex
 (PR https://github.com/Turfjs/turf/pull/897 - Contributor @stebogit @DenisCarriere)
-- Fixes `@turf/boolean-disjoint` incorrect behaviour where a poly was completely contained within another poly
+- Fixes [`@turf/boolean-disjoint`][boolean-disjoint] incorrect behaviour where a poly was completely contained within another poly
 (PR https://github.com/Turfjs/turf/pull/908 - Contributor @rowanwins)
-- Fixes `@turf/simplify` process pending on particular geometries, applying `@turf/clean-coords` to input.
+- Fixes [`@turf/simplify`][simplify] process pending on particular geometries, applying [`@turf/clean-coords`][clean-coords] to input.
 (PR https://github.com/Turfjs/turf/pull/903 - Contributor @stebogit @DenisCarriere)
-- Fixes `boolean` properties not being translated with `@turf/clone`
+- Fixes `boolean` properties not being translated with [`@turf/clone`][clone]
 (PR https://github.com/Turfjs/turf/pull/909 - Contributor @DenisCarriere)
-- Fixes `@turf/boolean-contains` incorrect output when checking two polygons 
+- Fixes [`@turf/boolean-contains`][boolean-contains] incorrect output when checking two polygons
 (PR https://github.com/Turfjs/turf/pull/923 - Contributor @rowanwins @DenisCarriere)
 
 
 Twitter: <Tweet>
-CC: @Turfjs/ownership 
+CC: @Turfjs/ownership
 
+
+[point-to-line-distance]: https://github.com/Turfjs/turf/tree/master/packages/turf-point-to-line-distance
```

Comment:
Cool!! I didn't know about these links in markdown! 👍

Reply:
😄 Yea they are pretty handy, also they don't show up in the final Markdown document, so I just copied the entire TurfJS module list.

Makes cleaner Markdown docs